docs/source/conventions/index.rst

```diff
@@ -30,3 +30,9 @@ Repository Conventions
     :maxdepth: 2
 
     file_structure
+
+In NWChemEx the following convetions are followed:
+C++ header: file_name.hpp
+C++ source files: file_name.cpp
+CMake source files : filename.cmake 
+CMake Lists: CMakeLists.txt
```

Comment:
```suggestion
In NWChemEx the following convetions are followed:

- C++ header: file_name.hpp
- C++ source files: file_name.cpp
- CMake source files : filename.cmake 
- CMake Lists: CMakeLists.txt
```

The formatting needs to be corrected to be a proper list. But I would move the information intended here to the files about the particular languages, e.g. move the note on C++ file extensions to [`cxx_conventions.rst`](https://github.com/NWChemEx/.github/blob/cc42c68e89c14c76112210ae467351d78fd8374a/docs/source/conventions/cxx_conventions.rst). Or the full section can be moved to [file_structure.rst](https://github.com/NWChemEx/.github/blob/cc42c68e89c14c76112210ae467351d78fd8374a/docs/source/conventions/file_structure.rst), maybe.